engine/src/main/java/org/terasology/rendering/dag/nodes/DownSampleSceneAndUpdateExposureNode.java

```diff
@@ -45,8 +49,19 @@
  * TODO: Add node of this diagram
  */
 public class DownSampleSceneAndUpdateExposureNode extends AbstractNode {
+    public static final ResourceUrn SCENE_16_URN = new ResourceUrn("engine:scene16");
+    public static final ResourceUrn SCENE_8_URN = new ResourceUrn("engine:scene8");
+    public static final ResourceUrn SCENE_4_URN = new ResourceUrn("engine:scene4");
+    public static final ResourceUrn SCENE_2_URN = new ResourceUrn("engine:scene2");
+    public static final ResourceUrn SCENE_1_URN = new ResourceUrn("engine:scene1");
+    public static final ResourceUrn SCENE_PRE_POST_URN = new ResourceUrn("engine:scenePrePost");
+
     private static final Logger logger = LoggerFactory.getLogger(DownSampleSceneAndUpdateExposureNode.class);
 
+    private PBO frontReadbackPBO;   // PBOs are 1x1 pixels buffers used to read GPU data back
+    private PBO backReadbackPBO;    // into the CPU. This data is then used in the context of
+    private PBO currentReadbackPBO; // eye adaptation.
```

Comment:
That been said, let's follow the example of the GBuffers and let's be more explicit about what are they for: let's rename them writeOnlyPBO and readOnlyPBO and let's have the appropriate functionality to swap them.